### PR TITLE
PHP Unit Testing

### PR DESCRIPTION
**Pull request by adamzero1**
Run PHPUnit tests, leveraging Nx to determine which tests need to be ran.

### CODE REVIEW

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          echo "Affected Projects: $(jq .projects ${AFFECTED_OUTPUT})"
+      
+      # Warden / Den stuff
+      - name: Checkout Den Repo
+        uses: actions/checkout@v3
+        with:
+          repository: swiftotter/den
+          path: den
+
+      - name: Init / Configure Den
```
Comment (Vinai) [inline .github/workflows/unit-tests.yml:105]: Would it make sense to switch to warden already, as den is kind of deprecated?

**Review by Vinai — COMMENTED** [review]
LGTM, only wondering if it makes sense to switch to warden already.

File: .github/workflows/unit-tests.yml (hunk 1 above)
Comment (damienwebdev) [reply .github/workflows/unit-tests.yml:105]: @Vinai do you mean the reverse? Den is the new, warden is the old.

Comment (sprankhub) [reply .github/workflows/unit-tests.yml:105]: No, @Vinai is correct: https://github.com/orgs/wardenenv/discussions/556

Comment (damienwebdev) [reply .github/workflows/unit-tests.yml:105]: What a confusing situation.

Comment (adamzero1) [reply .github/workflows/unit-tests.yml:105]: I know, haha never mind.
This should be good now. 
I've double checked and it appears to be working.

**Comment by adamzero1** [comment]
Hi all,
Just wondering what is left before we can merge this please?

**Review by Vinai — APPROVED** [review]
LGT

**Pull request merged**